DOCS/man/vo.rst

```diff
@@ -363,6 +363,76 @@ Available video output drivers are:
     ``--vo-tct-256=<yes|no>`` (default: no)
         Use 256 colors - for terminals which don't support true color.
 
+``sixel``
+    Sixel graphics video output driver based on libsixel that works on a
+    console that has sixel graphics enabled such as ``xterm`` or ``mlterm``.
+    Additionally some terminals have limitation on the dimensions, so may
+    not display images bigger than 1000x1000 for example. Make sure that
+    ``img2sixel`` can display images of the corresponding resolution.
+
+    Since mpv 0.30.0, you may need to use ``--profile=sw-fast`` to get decent
```

Comment:
The first part doesn't make sense here since vo_sixel does not exist in mpv 0.30.0.

Reply:
Okay I will just keep the `You may need to use --profile=sw-fast` part without the mpv version part.